frontend/src/components/GrantAccess.vue

```diff
@@ -94,6 +101,27 @@ function toRequestPayload(formData: any) {
 function statusSelected(evt: any) {
     formData.value.role = JSON.parse(evt.target.value);
 }
+
+async function getForestClientNumber(evt: string) {
+    loading.value = true;
+    try {
+        forestClient.value = (await forestClientNumberApi.search(evt))
+            .data as FamForestClient[];
+    } catch (err: any) {
+        return Promise.reject(err);
+    } finally {
+        loading.value = false;
+    }
```

Comment:
Maybe the frontend id done in certain way that I forgot already, but seeing catching an error and return `return Promise.reject(err);` seems a bit strange as the api should already be returned something equivalent as Promise.reject(). Just wondering here.
Also, in catch block, why isn't it set `loading.value` to `false`?

Reply:
It is the same logic as in `grantAccess()` method. Should it be changed in both cases? For the second question, the catch block doesn't need to set the `loading.value` because it is already in the finally block as it needs to be set to `false` either for successful calls as for errored ones.

Reply:
I would suggest please find out why you need to do nothing but only to return `Promise.reject(err)`, maybe there were already some certain way how it handle the error at frontend (but I kind of doubt that)... 
Feel feel to leave it as is if you could not find out the behavior and if we don't returns `Promise.reject(err))` does give you problem, but could you at next frontend ticket when you work on also try to see if this can be fixed all together? Thank you!

Sorry, I eyes are too big to see `finally` :P Thanks for pointing that out.

Reply:
I'll leave as is for now and work on that in the future as suggested.